### ISSUE TITLE
Flaky Apple Game Center test

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-server#8057]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description
Apple Game Center auth adapter tests fail sometimes:

```
1) Apple Game Center Auth adapter validateAuthData invalid signature id
- Expected a promise to be rejected with ParseError: 141 Apple Game Center - invalid signature but it was rejected with Error: error:09091064:PEM routines:PEM_read_bio_ex:bad base64 decode.
```

### Steps to reproduce
https://github.com/parse-community/parse-server/runs/6940095259?check_suite_focus=true#step:8:4251

### Actual Outcome
Test fails sometimes

### Expected Outcome
Test should pass

### Environment

Server
- Parse Server version: `5.2.2`

**Comment by mtrezza** [comment]
Closing, not observed anymore

**Comment by mtrezza** [comment]
Observed again

**Comment by dblythy** [comment]
Still observed, dammit

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0-alpha.22](https://github.com/parse-community/parse-server/releases/tag/5.3.0-alpha.22)

**Comment by dblythy** [comment]
Seems to have fixed the issue - ready to merge

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0-alpha.28](https://github.com/parse-community/parse-server/releases/tag/5.3.0-alpha.28)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/5.4.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0-alpha.1](https://github.com/parse-community/parse-server/releases/tag/5.4.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0](https://github.com/parse-community/parse-server/releases/tag/5.4.0)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0](https://github.com/parse-community/parse-server/releases/tag/5.4.0)
